module/users/helper.js

```diff
@@ -111,7 +111,15 @@ module.exports = class UsersHelper {
               duplicateProgram.endDate,
               userId
             );
-
```

Comment:
@VISHNUDAS-tunerlabs assuming you have validated with ImportFromLibrary API and normal flow as well please confirm